src/main/java/com/lorepo/icplayer/client/module/button/ButtonPresenter.java

```diff
@@ -270,4 +271,9 @@ public boolean isSelectable(boolean isTextToSpeechOn) {
 	public void onEventReceived(String eventName, HashMap<String, String> data) {
 		
 	}
+
```

Comment:
Robiąc tak psujesz funkcjonalność wszystkich innych button'ów napisanych w javie. Jeżeli jesteś w addonie button to powinieneś być w stanie z niego wyjść tab'em, bez potrzeby reset'u. isEneterable powinno mieć wartość true tylko dla ResetButton i tylko wtedy, gdy "confirm reset" jest ustawione na true, w każdym innym wypadku to powinno być false.